### PR TITLE
Move ezpdf to Vendor via Composer

### PR DESCRIPTION
**Pull request by Wakie87**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "require": {
 
         "smarty/smarty": "2.6.29",
-        "adodb/adodb-php": "5.20.2"
+        "adodb/adodb-php": "5.20.2",
+        "rospdf/pdf-php": "^0.12.22"
```
Comment (bradymiller) [inline composer.json]: This is probably a stupid questions, but what does the ^ mean?

Comment (Wakie87) [inline composer.json]: Caret#

The ^ operator behaves very similarly but it sticks closer to semantic versioning, and will always allow non-breaking updates. For example ^1.2.3 is equivalent to >=1.2.3 <2.0.0 as none of the releases until 2.0 should break backwards compatibility. For pre-1.0 versions it also acts with safety in mind and treats ^0.3 as >=0.3.0 <0.4.0.

This is the recommended operator for maximum interoperability when writing library code.

Example: ^1.2.3

[https://getcomposer.org/doc/articles/versions.md](https://getcomposer.org/doc/articles/versions.md)

**Comment by bradymiller** [comment]
How did you remove the samples from vendor? via build.xml?

File: composer.json (hunk 1 above)
Comment (bradymiller) [inline composer.json]: Doesn't that open the door though for developers (and production instances) to be using different versions (in the future when these are imported by developers)? Or am I being old too old fashioned :)

**Comment by bradymiller** [comment]
Did you test each of the pdf places in the codebase? (if so, I will then bring this into the codebase) (if weren't able to test them all, just let me know which ones are left to be tested)

**Comment by Wakie87** [comment]
Tested the prescription and Letter

File: composer.json (hunk 1 above)
Comment (bradymiller) [inline composer.json]: btw, thanks for the link. I now understand the ^ and ~
:)

**Comment by bradymiller** [comment]
I tested the other places which look good :)

**Comment by bradymiller** [comment]
I just brought this into the codebase. Thanks for the composer addition,
-brady

**Pull request closed without merging**